### PR TITLE
PWN-8689 - fix fees for claim on details

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-8689

## Description of Work

PWN-8689 - fix fees for claim on details

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt [hunk 1]
```diff
@@ -243,6 +243,12 @@ class RpcHistoryTransactionConverter(
             bundle?.fees?.arbiterFee,
             bundle?.fees?.gasFeeInToken
         )
+        val isFree = bundle?.compensationDeclineReason?.isEmpty() ?: false
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt:246]: isClaimFree?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt:246]: we already know that it is claim =[

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt:246]: we are in the parseWormholeReceive method, what should i think of? isReceiveFree? isBundleFree? isDeclineReasonFree? I couldn't understand it with the context provided, sorry
Damn, and which dog did bite you today?

Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt:246]: isTransactionWasFreeForClaimThatWasDoneInEthereum

**Review by eduardmaximovich — APPROVED** [review]

**Pull request merged**